src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -155,6 +155,7 @@ public function load(array $configs, ContainerBuilder $container): void
         }
 
         $container->setParameter('security.authentication.hide_user_not_found', $config['hide_user_not_found']);
+        $container->setParameter('security.authentication.show_account_status_exceptions', $config['show_account_status_exceptions']);
```

Comment:
```suggestion
        $container->setParameter('.security.authentication.show_account_status_exceptions', $config['show_account_status_exceptions']);
```

So that the parameter is compile-time only.